cluster-provision/k8s/1.25/k8s_provision.sh

```diff
@@ -146,7 +146,6 @@ plugins:
       usernames: []
       runtimeClasses: []
       # Hopefuly this will not be needed in future. Add your favorite namespace to be ignored and your operator not broken
-      # You also need to modify psa.sh
```

Comment:
We need to change the above enforcement level.

Reply:
@xpivarc you mean `enforce: "privileged"`? to which level?

Reply:
Yes, it needs to be changed to "restricted" as audit

Reply:
@xpivarc Thanks. Its difficult to find docs that explain these labels, had to go deep to [sig-auth](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2579-psp-replacement#summary).

Reply:
fyi there is nice doc https://kubernetes.io/docs/concepts/security/pod-security-standards/

Reply:
Yeah I saw it but it doesn't explain the meaning of `enforce, audit, warn`

Reply:
Oh, you are right. The bits are not on one page https://kubernetes.io/docs/concepts/security/pod-security-admission/